components/network/quagga/Makefile

```diff
@@ -0,0 +1,86 @@
+#
+# CDDL HEADER START
+#
+# The contents of this file are subject to the terms of the
+# Common Development and Distribution License (the "License").
+# You may not use this file except in compliance with the License.
+#
+# You can obtain a copy of the license at usr/src/OPENSOLARIS.LICENSE
+# or http://www.opensolaris.org/os/licensing.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+#
+# When distributing Covered Code, include this CDDL HEADER in each
+# file and include the License file at usr/src/OPENSOLARIS.LICENSE.
+# If applicable, add the following below this CDDL HEADER, with the
+# fields enclosed by brackets "[]" replaced with your own identifying
+# information: Portions Copyright [yyyy] [name of copyright owner]
+#
+# CDDL HEADER END
+#
+
+#
+# Copyright (c) 2006, 2019, Oracle and/or its affiliates. All rights reserved.
+# Copyright (c) 2020, Olaf Bohlen <olbohlen@eenfach.de>
+#
+BUILD_BITS= 32
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		quagga
+COMPONENT_VERSION=	0.99.19
+COMPONENT_REVISION=     1
```

Comment:
COMPONENT_REVISION should be not set for new components.

Reply:
fixed